packages/bpk-component-flare/src/BpkFlareBar.js

```diff
@@ -0,0 +1,78 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2016-2019 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import { cssModules } from 'bpk-react-utils';
+
+import STYLES from './bpk-flare-bar.scss';
+import Pointer from './__generated__/js/pointer';
+import CornerRadius from './__generated__/js/corner-radius';
+
+const getClassName = cssModules(STYLES);
+
+const BpkFlareBar = props => {
+  const { className, svgClassName, rounded, ...rest } = props;
+
+  const classNames = [getClassName('bpk-flare-bar__container')];
+  if (className) {
+    classNames.push(className);
+  }
+
+  const curveClassNames = [getClassName('bpk-flare-bar__curve')];
+  const leftCornerRadiusClassNames = [
+    getClassName('bpk-flare-bar__rounded-corner'),
+  ];
+  const rightCornerRadiusClassNames = [
+    getClassName('bpk-flare-bar__rounded-corner'),
+  ];
+  rightCornerRadiusClassNames.push(
```

Comment:
Could you merge this and the declaration above to be something like
```suggestion
  const rightCornerRadiusClassNames = [
    getClassName('bpk-flare-bar__rounded-corner'),
    getClassName('bpk-flare-bar__rounded-corner--trailing'),
  ];
```